blogposts/gophercon-2019-detecting-incompatible-api-changes.md

```diff
@@ -325,7 +325,7 @@ But you can change a pointer method to a value method. That works because the po
 
 How does the client code work with the new version of the package, where P is a value method? Go dereferences the pointer to get a value, then calls the method on that value.
 
-There’s a deliberate asymmetry in the language here. You can always call a value method on a pointer, because you can always dereference a pointer to get a value.
+there's a deliberate asymmetry in the language here. You can always call a value method on a pointer, because you can always dereference a pointer to get a value.
```

Comment:
```suggestion
There's a deliberate asymmetry in the language here. You can always call a value method on a pointer, because you can always dereference a pointer to get a value.
```